lib/geordi/cucumber.rb

```diff
@@ -108,6 +111,15 @@ def command_line_options
         # last arg. Do it manually here.
         last_arg = argv.last
         args << last_arg if (last_arg && last_arg.start_with?('-'))
+
+        args.map do |option|
+          if option.match?(/\S\s\S/)
+            # put options with a whitespace in quotes. Relevant for negated tag options ('not @tag')
```

Comment:
What is `option` when we're calling `--tags='not @tag'`? Why do we need to wrap something in quotes?

Please rename `args` to `options`.

Reply:
So we're 
1. separating arguments and options in the cucumber command file
2. merging them back into a flat array in `Geordi::Cucumber.run`
3. separating them again in `Geordi::Cucumber#command_line_options`
4. and merging them again in `Geordi::Cucumber#parallel_execution_command`??!

Could you please fix this?